README.md

```diff
@@ -28,15 +28,22 @@ brew install homebrew/cask/docker homebrew/cask/xquartz socat dtc
 If you are building CHERI on a Debian/Ubuntu-based machine, please install the following packages:
 
 ```shell
-apt install autoconf automake libtool pkg-config clang bison cmake ninja-build samba flex texinfo time libglib2.0-dev libpixman-1-dev libarchive-dev libarchive-tools libbz2-dev libattr1-dev libcap-ng-dev
+apt install autoconf automake libtool pkg-config clang bison cmake mercurial ninja-build samba flex texinfo time libglib2.0-dev libpixman-1-dev libarchive-dev libarchive-tools libbz2-dev libattr1-dev libcap-ng-dev
 ```
 
 Older versions of Ubuntu may report errors when trying to install `libarchive-tools`. In this case try using `apt install bsdtar` instead.
 #### RHEL/Fedora
 If you are building CHERI on a RHEL/Fedora-based machine, please install the following packages:
 
 ```shell
-dnf install libtool clang-devel bison cmake ninja-build samba flex texinfo glib2-devel pixman-devel libarchive-devel bsdtar bzip2-devel libattr-devel libcap-ng-devel expat-devel
+dnf install libtool clang-devel bison cmake mercurial ninja-build samba flex texinfo glib2-devel pixman-devel libarchive-devel bsdtar bzip2-devel libattr-devel libcap-ng-devel expat-devel
+```
+    
+#### FreeBSD
+If you are building CHERI on a RHEL/Fedora-based machine, please install the following packages:
```

Comment:
```suggestion
If you are building CHERI on a FreeBSD machine, please install the following packages:
```
The "please install the following packages" phrasing followed by an install command used throughout this section is generally a bit odd.